protocol/river-status-unstable-v1.xml

```diff
@@ -16,7 +16,7 @@
     OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
   </copyright>
 
-  <interface name="zriver_status_manager_v1" version="1">
+  <interface name="zriver_status_manager_v2" version="2">
```

Comment:
You don't need to update the `_v1` in the name of the interface. It's still `_v1`, just version 2 of it. Weird, I know.

Reply:
Thanks, I'm so confused by protocols versionning